services/net/contentmigration/sources/oracle/MigrationSourceContext.cs

```diff
@@ -74,6 +79,19 @@ protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
     /// <param name="modelBuilder"></param>
     protected override void OnModelCreating(ModelBuilder modelBuilder)
     {
+        modelBuilder.HasDefaultSchema(Schema);
+
+        if (string.IsNullOrWhiteSpace(Schema))
```

Comment:
Change to a simpler easier to maintain solution by using string interpolation.  Having multiple sql statements to maintain will always result in regression errors.

`.ToSqlQuery($"WITH ATN AS (SELECT * from {(!String.IsNullOrEmpty(Schema) ? $"{Schema}." : "")}.NEWS_ITEMS UNION ALL SELECT * from  {(!String.IsNullOrEmpty(Schema) ? $"{Schema}." : "")}.HNEWS_ITEMS) SELECT * FROM ATN")

Reply:
Or cleaner would be
`var schema = (!String.IsNullOrEmpty(Schema) ? $"{Schema}." : "";
`.ToSqlQuery($"WITH ATN AS (SELECT * from {schema}.NEWS_ITEMS UNION ALL SELECT * from  {schema}.HNEWS_ITEMS) SELECT * FROM ATN"`

Reply:
option 2 is better for sure.